scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -111,6 +123,8 @@ def ttest_ci(*args, **kwargs):
     (stats.alexandergovern, tuple(), {}, 2, 2, False,
      lambda res: (res.statistic, res.pvalue)),
     (stats.combine_pvalues, tuple(), {}, 1, 2, False, None),
+    (xp_mean_1samp, tuple(), dict(), 1, 1, False, lambda x: (x,)),
+    (xp_mean_2samp, tuple(), dict(), 2, 1, True, lambda x: (x,)),
```

Comment:
Most `scipy.stats` functions use the `_axis_nan_policy` decorator to implement `nan_policy`, `keepdims`, and tuple `axis`. I've implemented all these features natively for improved performance (e.g. `nan_policy='omit'` would otherwise loop over each slice), and the function still passes all the tests, which are quite stringent. So if you don't want to write tests with `hypothesis`, I'm still pretty comfortable with this.